assets/js/blocks/product-categories/block.js

```diff
@@ -115,43 +104,26 @@ class ProductCategoriesBlock extends Component {
 										{ this.renderOptions( categories ) }
 									</select>
 								</div>
-								<IconButton
+								<button
+									type="button"
+									className="components-button components-icon-button"
```

Comment:
I'm ok doing that in a follow-up, but it looks like these classes use styles from wp-admin stylesheets (`css/dist/components/style.css`) could we avoid importing that file and instead creating the new styles required for this button in `product-categories/style.scss`?

Reply:
👍